docs/build.rst

```diff
@@ -85,7 +86,7 @@ Option                  Via setup.py                Via environment variable
 ======================= =========================== ===========================
 Custom path to HDF5     ``--hdf5=/path/to/hdf5``    ``HDF5_DIR=/path/to/hdf5``
 Force HDF5 version      ``--hdf5-version=X.Y.Z``    ``HDF5_VERSION=X.Y.Z``
-Enable MPI mode         ``--mpi``                   (none)
+Enable MPI mode         ``--mpi``                   ``HDF5_PARALLEL=ON``
```

Comment:
Actually "parallel" is correct in hdf5 notation (see hdf5 configure and h5p namings). The --mpi option is actually not precise enough when talking about mpi io based file operations (one can e.g. also do mpi based, independent posix operations to individual files).